getting-started-gradle/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.acme</groupId>
+    <artifactId>getting-started-gradle</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
We can drop the pom here. We want it to be a purely Gradle project. We can also remove the Maven wrapper from this directory if it's still there (haven't checked).

Reply:
Sure